### PR TITLE
Fix warnings for Rekordbox library feature

### PR DESCRIPTION
**Pull request by uklotzde**

### CODE REVIEW

File: src/library/rekordbox/rekordboxfeature.cpp [hunk 1]
```diff
@@ -663,7 +663,7 @@ void clearDeviceTables(QSqlDatabase& database, TreeItem* child) {
     transaction.commit();
 }
 
-void setHotCue(TrackPointer track, double position, int id, QString label, int colorCode, int colorRed, int colorGreen, int colorBlue) {
+void setHotCue(TrackPointer track, double position, int id, QString label, int /*colorCode*/, int /*colorRed*/, int /*colorGreen*/, int /*colorBlue*/) {
```
Comment (daschuer) [inline src/library/rekordbox/rekordboxfeature.cpp]: Normally we use Q_UNUSED ... Next time.

**Review by daschuer — APPROVED** [review]

**Comment by daschuer** [comment]
Waiting for CI

**Comment by Holzhaus** [comment]
Why did you close this?

**Comment by uklotzde** [comment]
Ups, I accidentally deleted the branch while cleaning up my repo.

**Comment by Holzhaus** [comment]
$ grep -Po "^src/.*\K\[-W.*\]" warnings_master.log | sort | uniq -c
         18 [-Wdeprecated-declarations]
         13 [-Wrestrict]
         10 [-Wswitch]
          6 [-Wunused-parameter]
    $ grep -Po "^src/.*\K\[-W.*\]" warnings_rekordbox.log | sort | uniq -c
         18 [-Wdeprecated-declarations]
         13 [-Wrestrict]
         10 [-Wswitch]
          2 [-Wunused-parameter]

**Comment by Holzhaus** [comment]
The `-Wswitch` warnings all complain about unhandled enum values in `src/library/rekordbox/rekordbox_pdb.cpp`, line 952. Might be a good idea to add a default case (maybe containing a `qDebug()` statement?`.

The two unused parameter warnings are from `src/library/rekordbox/rekordbox_anlz.cpp:7` and `src/library/rekordbox/rekordbox_pdb.cpp:7` and referring to parameter `p__root`.

**Comment by Holzhaus** [comment]
Nice! LGTM. Would be nice if you could fix the remaining warnings in the rekordbox code, too.

**Comment by uklotzde** [comment]
Fixing warnings in generated code (_anlz/_pdb) manually doesn't help in the long term.

**Comment by daschuer** [comment]
Ok so we can merge this now.

**Pull request merged**